tests/loggers/test_base.py

```diff
@@ -290,3 +291,37 @@ def log_hyperparams(self, params):
     }
     logger.log_hyperparams(Namespace(**np_params))
     assert logger.logged_params == sanitized_params
+
+
+@pytest.mark.parametrize("logger", [True, False])
+def test_log_hyperparams_being_called(tmpdir, logger):
+
+    class TestLogger(DummyLogger):
+
+        def __init__(self):
+            super().__init__()
+            self.log_hyperparams_called = False
+
+        def log_hyperparams(self, *args, **kwargs):
+            self.log_hyperparams_called = True
```

Comment:
can you patch the method in the logger instead?

With the decorator
```python
@patch("pytorch_lightning.loggers.tensorboard.TensorBoard.log_hyperparams")
```
or passing a mock as the logger object

```python
logger_mock = Mock()
trainer = Trainer(logger=logger_mock, ...)
trainer.fit(...)

mock.log_hyperparams.assert_not_called()
```

Reply:
@awaelchli Updated!